<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>uberqt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
	background-image: url(:/background/theme/background.svg);
	background-color: qradialgradient(spread:pad, cx:0.48, cy:0.533884, radius:0.597, fx:0.237, fy:0.213029, stop:0.14375 rgba(2, 28, 72, 255), stop:0.638535 rgba(1, 13, 34, 255), stop:1 rgba(0, 0, 0, 255))
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
      <item>
       <widget class="ClockContainer" name="clockContainer" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="optionsFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
         <item>
          <widget class="QPushButton" name="exit">
           <property name="text">
            <string>Quit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fullscreen">
           <property name="text">
            <string>Fullscreen</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
           <item>
            <widget class="QPushButton" name="zoomIn">
             <property name="text">
              <string>Zoom In</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="zoomOut">
             <property name="text">
              <string>Zoom Out</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QToolBox" name="tools">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <widget class="QWidget" name="boxAlarm">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>374</width>
              <height>356</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <attribute name="label">
             <string>Alarm</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QListView" name="listView"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
               <property name="margin">
                <number>5</number>
               </property>
               <item>
                <widget class="QPushButton" name="ucNew">
                 <property name="text">
                  <string>New</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="ucDelete">
                 <property name="text">
                  <string>Delete</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="ucEdit">
                 <property name="text">
                  <string>Edit</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="1" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Wakup Time</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QTimeEdit" name="ucTime">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Program</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="ucProgram">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="boxThemes">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>374</width>
              <height>356</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Theme</string>
            </attribute>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="preferences">
           <property name="text">
            <string>Clock Preferences</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClockContainer</class>
   <extends>QWidget</extends>
   <header>clockcontainer.h</header>
   <container>1</container>
   <slots>
    <signal>clicked(QEvent)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../clock.qrc"/>
 </resources>
 <connections/>
</ui>
